### PR TITLE
Upgrade to 0.14.1

### PR DESCRIPTION
**Pull request by Erick555**
Fixes https://github.com/flathub/im.riot.Riot/issues/12

### CODE REVIEW

File: im.riot.Riot.appdata.xml [hunk 1]
```diff
@@ -26,6 +26,7 @@
     </screenshot>
   </screenshots>
   <releases>
+    <release version="0.14.1" date="2018-04-22"/>
```
Comment (barthalion) [inline im.riot.Riot.appdata.xml]: should be 2018-04-12

**Comment by barthalion** [comment]
bot, build

File: im.riot.Riot.json [hunk 2]
```diff
@@ -59,8 +59,8 @@
                 },
                 {
                     "type": "archive",
-                    "url": "https://github.com/vector-im/riot-web/releases/download/v0.14.0/riot-v0.14.0.tar.gz",
-                    "sha256": "51efd76a2badcce0bf339111c3d3ecf50c74e8bb555d0f36c653c2d746435669"
+                    "url": "https://github.com/vector-im/riot-web/releases/download/v0.14.0/riot-v0.14.1.tar.gz",
```
Comment (barthalion) [inline im.riot.Riot.json]: bad url (you missed one 0 before the filename)

File: im.riot.Riot.appdata.xml (hunk 1 above)
Comment (Erick555) [reply im.riot.Riot.appdata.xml]: Sorry, I misunderstood this. Fixed

File: im.riot.Riot.json (hunk 2 above)
Comment (Erick555) [reply im.riot.Riot.json]: Fixed, sorry.

**Comment by nedrichards** [comment]
bot, build

**Comment by barthalion** [comment]
Thanks!

**Pull request merged**